auth-api/src/auth_api/services/affiliation_invitation.py

```diff
@@ -207,6 +208,19 @@ def _get_invitation_email(affiliation_invitation_info: Dict,
 
         return affiliation_invitation_info.get('recipientEmail', None)
 
+    @staticmethod
+    def _get_org_id_from_org_uuid(to_org_uuid):
+        try:
```

Comment:
I think you're fine without a try raise here - but that's minor  - this is already in the feature branch - approving